src/state/selectors.ts

```diff
@@ -61,7 +61,7 @@ export const getIsBanned =
       RD.map(({ [buddyId]: buddy }) => {
         return buddy ? buddy.status === 'Banned' : false;
       }),
-      RD.getOrElse<unknown, boolean>(() => false),
+      RD.getOrElse<unknown, boolean>(() => true),
     );
```

Comment:
If buddy has no status it's banned. Is this okay? I have no idea about API, seems quite a wild?

Reply:
If we dont default to banned, then banned users messages will briefly show the yellow-unseen-messages-dot to the user before disappearing after this calculation is done.